diff -u pqm4_kyber512.old/kem.c pqm4_kyber512/kem.c
--- pqm4_kyber512.old/kem.c	2022-03-28 16:36:40.045941053 -0400
+++ pqm4_kyber512/kem.c		2022-03-11 13:38:07.166786818 -0500
@@ -1,4 +1,4 @@
-#include "api.h"
+#include "api_kyber.h"
 #include "indcpa.h"
 #include "params.h"
 #include "randombytes.h"
diff -u pqm4_kyber512.old/randombytes.c pqm4_kyber512/randombytes.c
--- pqm4_kyber512.old/randombytes.c	2022-03-28 16:36:40.037940919 -0400
+++ pqm4_kyber512/randombytes.c		2022-03-11 14:38:18.192717915 -0500
@@ -357,6 +357,10 @@
     /* Use WASI */
     return randombytes_wasi_randombytes(buf, n);
     #else
-# error "randombytes(...) is not supported on this platform"
+    #warning randombytes() is not supported on this platform!
+    for (size_t i = 0; i < n; i++) {
+         output[i] = (uint8_t)i;
+    }
+    return 0;
     #endif
 }
